### DISCUSSION TITLE
Ignore/mute notifications for pushes from `github-actions` user

### DISCUSSION THREAD

**Discussion by mrcnski** [lowlighter/metrics#1078] [Q&A] [answered]
Hi, thank you for this fantastic service, and also for being thorough in addressing issues. It is really great to see that.

The action works well for me - too well, actually, and now my GitHubs homepage is spammed with:

<img width="641" alt="Screen Shot 2022-05-26 at 10 53 30" src="https://user-images.githubusercontent.com/6035856/170525682-961b0f5a-8544-4f57-858b-8670dcdb1517.png">

etc. So I'm asking here in case anyone knows whether it's possible to disable these notifications just for this repo. If not, I'll have to lower the action frequency to once a day, which is not ideal but better than dealing with the spam.

**Comment by spenserblack** [comment, marked as answer]
Try changing your watch settings on [your profile repo](https://github.com/m-cat/m-cat).

**Reply by mrcnski** [reply]
Thanks for the reply! Do you know if there is a way to only unwatch a single user?

**Reply by spenserblack** [reply]
Not that I know of, but I think with a custom watch setting (maybe issues, PRs, and discussions only), you won't be *fully* watching, and commit notifications will be excluded. Haven't tried this out, though.

**Reply by mrcnski** [reply]
I'll give it a try.

**Reply by mrcnski** [reply]
It works, thanks!
